content/en/docs/how-tos/artifacts.md

```diff
@@ -0,0 +1,340 @@
+  format.
+
+- `finished.json`: the file uploaded by the either the `initupload` or `sidecar`
+  containers when the test finishes.
+
+- `podinfo.json`: a dump of the `Pod` and `Event` objects associated with the
+  test, also found in the "Pod" and "Events" sections of the main page.
+
+- `prowjob.json`: a dump of the `ProwJob` object, also found in the "Prow Job
+  YAML" link in the main page.
+
+- `sidecar-logs.json`: output of the `sidecar` container, which is responsible
+  for uploading the files in this directory.  It corresponds to the `sidecar`
+  section at the bottom of the main page.
+
+- `started.json`: file uploaded by the `initupload` _init_ container when the
+  test starts.
+
+### `ci-operator` Artifacts
+
+A test executed by `ci-operator` will also have a directory called `artifacts`
+among the files described in the previous section.  This is the first directory
+in the previous listing and has the following contents:
+
+{{< highlight bash >}}
+.
+├── build-logs
+│   ├── …
+│   └── src.log
+├── build-resources
+│   ├── builds.json
+│   ├── events.json
+│   ├── imagestreams.json
+│   ├── pods.json
+│   └── templateinstances.json
+├── test
+│   ├── artifacts
+│   │   ├── …
+│   │   └── junit.xml
+│   ├── build-log.txt
+│   ├── finished.json
+│   └── sidecar-logs.json
+├── ci-operator-step-graph.json
+├── ci-operator.log
+├── junit_operator.xml
+└── metadata.json
+{{< / highlight >}}
+
+The files immediately under the directory are not commonly useful to test
+authors, they are mostly meant for test platform engineers:
+
+- `ci-operator-step-graph.json`: this is a dump of every Kubernetes/OpenShift
+  object created by each step (n.b.: "step" here means `ci-operator` step, not a
+  step of a multi-stage test &mdash; the latter being a type of the former).
+
+- `ci-operator.log`: the `ci-operator` debug output.  It is a superset of
+  `build-log.txt`, in JSON format.  It can sometimes contain extra information
+  useful for understanding the main log.
+
+- `junit_operator.xml`: the source of the JUnit section in the main page.
+
+- `metadata.json`: data file consumed by
+  [`testgrid`](https://github.com/kubernetes/test-infra/tree/master/testgrid).
+
+The directories at the top level contain detailed information for objects
+created by the `ci-operator` test:
+
+- `build-logs`: this directory contains one file for each OpenShift build
+  created by the test containing the full log of the build.  `ci-operator` will
+  include these logs in its output in case of failure, but they are available
+  here directly as text files for all builds, even successful ones.
+
+- `build-resources`: this directory contains dumps of objects in the test
+  namespace at the time the test finished, as listed below.  Each file is named
+  after the type of object dumped.  Note that, since the test namespace can be
+  and is commonly shared by many tests, these dumps may contain objects which
```

Comment:
I do not get "test namespace can be and is commonly shared by many tests". Does it means that multi-stage tests run inside the same namespace or that multiple tests (different ProwJobs) could share the same namespace?

Reply:
https://docs.ci.openshift.org/docs/how-tos/interact-with-running-jobs/#how-and-where-do-the-tests-run